Makefile

```diff
@@ -47,10 +47,15 @@ dockertogoarch = $(patsubst arm/v7,arm,$(1))
 
 deploy: images
 
+lint: pkg/natdiscovery/proto/natdiscovery.pb.go
```

Comment:
for some reason this is not triggering in lint, and failing the check. :?

Reply:
ok, changing this to: 
`golangci-lint: pkg/natdiscovery/proto/natdiscovery.pb.go`

it seems to work